### PR TITLE
add low resolution tests

### PR DESCRIPTION
**Pull request by DeniseWorthen**
### Description of changes

- Adds  tests to replicate existing c96-1/4 deg tests (decomp, threads etc) at c96-1deg for the coupled model.

- Changes default to be a 1d run. Replaces 2d/3d/restart test with a 12h/1d/restart test. Restart test is currently implemented at c96mx025 resolution.

- Changes mediator restart directory to RESTART. Mediator restart ``case_name`` is changed from "ufs.s2s" to "ufs.cpld"

- Updates MOM6 and FV3 to match current ufs-weather

### Specific notes

Issues Fixed (include github issue #):

ufs-weather Issue [#222](https://github.com/ufs-community/ufs-weather-model/issues/222)
Issue #83 
Dependent PR #204 

Are changes expected to change answers?
- new baselines required for all tests
- tests which replicate current baselines (c384mx025) were shown to be b4b in PR #204

Specific changes:
- cold nems.configure files are removed
- cpld_control_c192 (c192mx050), cpld_ca (c96mx100 Cellular Automata) are added in addition to c96/mx100 version of existing tests. 
- All tests run with either ``fv3_conf/cpld_control_run.IN`` or ``fv3_conf/cpld_bmark_run.IN`` 

Testing performed:
- machines: hera,orion,dell-p3
- details:

Hashes used for testing:

- NEMS: f102e36
- CMEPS: 4d50adf
- FV3: b955f81
- MOM6: 7a76a13
- CICE: ac617cd
- WW3: 9c22b13
- FMS:f61416f
- stochastic_physics: 6eac84

Co-authored by:

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA @junwang-noaa 

I'm having two problems w/ the s2s_restart test and I need  a little help. I've set up  s2s_control (1day), s2_control_12h (12h) and s2s_restart (a 12h run, dependent to the 12h test). The s2s_restart test compares to the s2s_control baseline. 

1. If I try to create a baseline ( ./rt.sh -cefk > output 2>&1), the s2s_restart test tries to run. Since the restart test doesn't have a baseline, I thought that this test is skipped if the baseline is getting created. The run directory for this step is here:
/work/noaa/stmp/dworthen/stmp/dworthen/S2S_RT/rt_242741

2. The restart test in the above directory does seem to be copying in the right dependent run input (from ../s2s_control_12h). But the run fails with an error about time in the fv3_cap.F90: 

20201001 232020.424 WARNING          PET000 ESMCI_Clock.C:1705 ESMCI::Clock::validate() timeStep equals zero.
20201001 232020.433 ERROR            PET000 ESMCI_Clock.C:373 ESMCI::Clock::set() Wrong data value  - Internal subroutine call returned Error
20201001 232020.433 ERROR            PET000 ESMF_Clock.F90:1695 ESMF_ClockSet() Wrong data value  - Internal subroutine call returned Error
20201001 232020.436 ERROR            PET000 fv3_cap.F90:546 Wrong data value  - Passing error in return code
20201001 232020.436 INFO             PET000 Finalizing ESMF

I must have something wrong in the s2s_restart/model.configure or input.nml but I can't spot what it is.

The build directory is here: /work/noaa/marine/dworthen/S2S_lowres/tests

**Comment by junwang-noaa** [comment]
For 1), please make sure that the 5th column of s2s_restart test in rt.conf
is empty, it should not have "fv3".

I can't see your code, please open read permission:
Orion-login-3[13] junwang$ ls /work/noaa/marine/dworthen/S2S_lowres/tests
ls: cannot access /work/noaa/marine/dworthen/S2S_lowres/tests: Permission
denied

**Comment by DeniseWorthen** [comment]
I've changed the permissions.

I will remove the fv3; Thanks.

**Comment by MinsukJi-NOAA** [comment]
```
jminsuk@Orion-login-2:s2s_restart$ diff model_configure ../s2s_control_12h/model_configure 
21,22c21,22
< restart_interval:        24
< fhrot:                   12
---
> restart_interval:        12
> fhrot:                   0
jminsuk@Orion-login-2:s2s_restart$ diff input.nml ../s2s_control_12h/input.nml 
48c48
<        make_nh = .F.
---
>        make_nh = .T.
71c71
<        na_init = 0
---
>        na_init = 1
83c83
<        external_ic = .F.
---
>        external_ic = .T.
86,87c86,87
<        nggps_ic = .F.
<        mountain = .T.
---
>        nggps_ic = .T.
>        mountain = .F.
101c101
<        warm_start = .T.
---
>        warm_start = .F.
313c313
<          input_filename = 'r'
---
>          input_filename = 'n'
```

**Comment by MinsukJi-NOAA** [comment]
Since s2s_control_12h worked, I am comparing model_configure and input.nml between s2s_restart and s2s_control_12h

**Comment by MinsukJi-NOAA** [comment]
Denise, can you change the permission for log_orion.intel as well?

```
cd log_orion.intel/
-bash: cd: log_orion.intel/: Permission denied
```

**Comment by DeniseWorthen** [comment]
Geez I might have messed up. I changed the permissions but then I tested again with the fv3 removed from that column to see that it does fix the first problem (trying to run the restart test during baseline generation). 

I then started it up another time to verify that the restart test can run and compare. So I've changed the permissions again (maybe the newly generated files have the old permissions?). Now when I try to actually run the restart test (./rt.sh -efk >output 2>&1 &) I'm left only with the 2nd problem (some sort of clock issue where the run just fails).

**Comment by junwang-noaa** [comment]
Denise, please set FHMAX=24
in /work/noaa/marine/dworthen/S2S_lowres/tests/tests/s2s_restart

**Comment by DeniseWorthen** [comment]
Thanks both of you. The restart test is now running correctly during baseline creation and verification. It isn't passing yet though so I need to double check the settings (particularily MOM6, which we know needs a special setting for restart repro).

**Comment by DeniseWorthen** [comment]
I am not able to pass the restart test using c96-1deg. FV3 is not reproducing on restart for 5 fields: atmImp_Faxa_snow, atmImp_Sa_pbot, atmImp_Sa_shum, atmImp_Sa_tbot, atmImp_Sa_z. In all cases, there are 9 points on the FV3 tile3 that do not reproduce on restart.

**Comment by junwang-noaa** [comment]
You are using the non-fractional grid ICs, right?

**Comment by DeniseWorthen** [comment]
That is right. I am testing c96/mx100 in non-frac mode. The same test in frac-mode shows an additional 2 fields which do not reproduce: atmImp_Faxa_lwdn and atmImp_Faxa_lwnet. 

In both cases I'm reporting the field that is sent to the mediator (atmImp) from FV3 at restart.

**Comment by DeniseWorthen** [comment]
I now have a test branch for doing the restart test for non-frac c384mx025 using the new 1d/12h and 12 restart setup. 
Using the original input FV3 input (ie, not the newly generated C384mx025 input which is frac_grid capable),  I find that FV3 is reproducing on restart.

**Comment by DeniseWorthen** [comment]
This will also update NEMS to [f102e36](https://github.com/NOAA-EMC/NEMS/tree/develop). This should be OK since that the NEMS commit states it should happen after weather PR [213](https://github.com/ufs-community/ufs-weather-model/pull/213) which has been closed.

File: tests/tests/cpld_ca [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                   #RESTART/ufs.cpld.cpl.r.2016-10-04-00000.nc"
+
+export_fv3
+export_cpl
+
+export DO_CA=.T.
+export CA_SGS=.T.
+export CA_GLOBAL=.T.
+
+export FV3_RUN=cpld_control_run.IN
```
Comment (junwang-noaa) [inline tests/tests/cpld_ca:65]: It looks to me the input.nml for this test is still the default: export INPUT_NML=input.mom6_ccpp.nml.IN under parm and[ input.mom6_ccpp.nml.IN ](https://github.com/DeniseWorthen/ufs-s2s-model/blob/feature/rtres/parm/input.mom6_ccpp.nml.IN) still has:
do_ca          = .false.
       ca_sgs         = .false.
should it have: 
       do_ca          = @[DO_CA]
       ca_sgs         = @[CA_SGS]
       ca_global      = @[CA_GLOBAL]

**Comment by DeniseWorthen** [comment]
I will regenerate the CA baseline test.

I also have a fix for the FV3_RESTART_H issue and have already tested it. I will merge that fix and then re-run all the baselines.

**Comment by junwang-noaa** [comment]
Denise, thanks for making the code changes.

**Comment by DeniseWorthen** [comment]
All RTs have been re-run. My commit message on the last one mistakenly said 'orion logs'. It is the p3 logs.

**Review by junwang-noaa — APPROVED** [review]
This is a lot work in the two PRs (194 and 204). Thanks for the nice work!

**Review by JessicaMeixner-NOAA — APPROVED** [review]
Looks great! Thanks for adding the cpld CA case and in general just making everything cleaner.

**Pull request merged**